wingtips-core/src/main/java/com/nike/wingtips/Span.java

```diff
@@ -36,35 +39,39 @@
-    /** The name of the time-spent-in-nanos field when serializing to JSON (see {@link #toJSON()}. Corresponds to {@link #getTimeSpentNanos()}. */
-    public static final String TIME_SPENT_NANOS_FIELD = "timeSpentNanos";
+    /** The name of the start-time-in-epoch-micros field when serializing/deserializing to/from JSON (see {@link #toJSON()} and {@link #fromJSON(String)}). Corresponds to {@link #getSpanStartTimeNanos()}. */
+    public static final String START_TIME_EPOCH_MICROS_FIELD = "startTimeEpochMicros";
+    /** The name of the duration-in-nanoseconds field when serializing to JSON (see {@link #toJSON()}. Corresponds to {@link #getDurationNanos()}. */
+    public static final String DURATION_NANOS_FIELD = "durationNanos";
 
     private final String traceId;
     private final String spanId;
     private final String parentSpanId;
     private final String spanName;
     private final boolean sampleable;
     private final String userId;
+    private final long spanStartTimeEpochMicros;
     private final long spanStartTimeNanos;
-    private Long spanEndTimeNanos;
 
-    private Long timeSpentNanos;
+    private Long durationNanos;
 
     private String cachedJsonRepresentation;
 
     private String cachedKeyValueRepresentation;
 
     /**
-     * The full constructor allowing you to set all fields representing a span's state. The traceId, spanId, and spanName fields cannot be null (an {@link IllegalArgumentException}
-     * will be thrown if any are null). This method is public to allow full flexibility, however in practice you should use one of the provided
-     * {@link #generateRootSpanForNewTrace(String)} or {@link #generateChildSpan(String)} helper methods, or the {@link Builder} if you need more flexibility than
-     * the helper methods.
+     * <p>
+     *  The full constructor allowing you to set all fields representing a span's state. The traceId, spanId, and spanName fields cannot be null (an
+     *  {@link IllegalArgumentException} will be thrown if any are null). This method is public to allow full flexibility, however in practice you should use
+     *  one of the provided {@link #generateRootSpanForNewTrace(String)} or {@link #generateChildSpan(String)} helper methods, or the {@link Builder} if you
+     *  need more flexibility than the helper methods.
+     * </p>
+     * WARNING: When deserializing a span that was initially started on a different JVM, you *must* pass in null for {@code spanStartTimeNanos}. Otherwise
+     * timing will be completely broken since {@link System#nanoTime()} is not comparable across JVMs.
      */
-    public Span(String traceId, String parentSpanId, String spanId, String spanName, boolean sampleable, String userId, long spanStartTimeNanos, Long spanEndTimeNanos) {
+    public Span(String traceId, String parentSpanId, String spanId, String spanName, boolean sampleable, String userId,
+                long spanStartTimeEpochMicros, Long spanStartTimeNanos, Long durationNanos
```

Comment:
This is partly why I wanted to stay sub-1.0 to start with. I'll probably bump it to 0.10.0, and I'll call out the breaking change in the change notes. But since we're sub-1.0 I don't want to start adding unnecessary cruft just to stay backwards compatible.

There will be some more minor breaking changes coming in the next few PRs too. :)